data/org.freedesktop.fwupd.metainfo.xml

```diff
@@ -38,6 +38,414 @@
     <binary>fwupdtool</binary>
   </provides>
   <releases>
+    <release version="2.0.0" date="2024-08-27">
+      <description>
+        <p>
+          This new major release adds the following project-wide features:
+        </p>
```

Comment:
need to make noise about the ABI change

Reply:
I've added `Remove deprecated API from libfwupd, bumping soname of libfwupd` further down, but maybe that's not explicit enough. How about:

`This new major release beaks libfwupd ABI and API, to add the following project-wide features`

Reply:
Yes.

Reply:
Cool, thanks. Feel free to add fixups to that branch  - i think we should aim to get 2.0.0 by the end of next week or we're never going to get there. I'm under a bit of pressure from various companies to start tagging releases from the main branch again. I think we can make 1.9.x a "bugfixes only" policy then.

Reply:
> i think we should aim to get 2.0.0 by the end of next week or we're never going to get there.

The one thing I would really like to see before tagging 2.0.0 is the emulation capture and use for non-USB working.  I would feel a lot more comfortable with living with all the symbols if we know there are no gaps with that.

Reply:
Me also; I've got a couple of PRs to do to get that over the line. [ssss](https://github.com/fwupd/fwupd/pull/7501) would be the first that list.